.github/workflows/kind.yaml

```diff
@@ -98,6 +100,14 @@ jobs:
           kubectl apply -n external-targets -f .github/external-targets/nginx.yaml
           kubectl rollout status -n external-targets ds/nginx
 
+      - name: Set up node local DNS
+        run: |
+          curl -sSfL -o .github/node-local-dns/node-local-dns.yaml https://raw.githubusercontent.com/cilium/cilium/main/examples/kubernetes-local-redirect/node-local-dns.yaml
+          kubedns=$(kubectl get svc kube-dns -n kube-system -o jsonpath={.spec.clusterIP}) && sed -i "s/__PILLAR__DNS__SERVER__/$kubedns/g;" .github/node-local-dns/node-local-dns.yaml
+          kubectl apply -k .github/node-local-dns
+          kubectl apply -f https://raw.githubusercontent.com/cilium/cilium/main/examples/kubernetes-local-redirect/node-local-dns-lrp.yaml
+          kubectl rollout status -n kube-system ds/node-local-dns
```

Comment:
Maybe we are better off just checking in `node-local-dns.yaml` to this repo instead to prevent breaking changes to it in cilium/cilium and also it will prevent random internet flakes downloading the file

Reply:
Thank you for your comment! I checked in `node-local-dns.yaml` and `node-local-dns-lrp.yaml` to `.github/node-local-dns`